api/v1alpha1/base_types.go

```diff
@@ -279,6 +279,14 @@ type RestoreSource struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	TargetRecoveryTime *metav1.Time `json:"targetRecoveryTime,omitempty" webhook:"inmutable"`
+	// FileName is the file within the source to be restored.
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	FileName *string `json:"fileName,omitempty" webhook:"inmutableinit"`
```

Comment:
Is there any way we could integrate the physical backup feature with the PITR? This is quite recent, see the docs:
- https://github.com/mariadb-operator/mariadb-operator/blob/main/docs/BACKUP.md

If that's not the case, could we expand the `FileName` explanation a bit further to indicate that this is supposed to be used only in physical backups.

We don't necessarily have to do it in this PR, I can take a look as part of a different PR.

Reply:
Hey @mmontes11, since mariadbbackup is a physical backup solution, it does not support PITR. But here I think we use PITR in the wrong sense. PITR means restoring the database to the exact time, which can be achieved using binary logs. But here we just use the time to find the closest backup file and import the entire backup. The same can be done for the mariadbbackup but we should not call this PITR. this may lead to misunderstanding

Reply:
right, understood. PITR implies the entire state of the database not just the data (logical state). I think we should rename it to "target recovery time" ?

Reply:
Thanks a lot for the suggestion, documentation updated accordingly:
https://github.com/mariadb-operator/mariadb-operator/blob/main/docs/BACKUP.md